purchase_ext_sst/models/purchase_order.py

```diff
@@ -192,6 +192,8 @@ def button_confirm(self):
                                   'default guest user.'))
             if not purchase_order.date_planned:
```

Comment:
The onchange method does not update the `date_planned` of the order

Reply:
@yostashiro , I think this part is needed.
When user do not input the `date_planned` of the order, it will be empty when the order is confirmed.

Reply:
@TimLai125 In that case, can we add a default value (today) to date_planned field rather than having the logic redundantly. I'm not sure if onchange is triggered by the default value.  If it is not, then we may make line date_planned a stored computed field.

Reply:
> In that case, can we add a default value (today) to date_planned field rather than having the logic redundantly


But the order may not be confirmed in the same day of the creation.